docs/src/content/docs/guides/outgoing-grant-no-recipients.mdx

```diff
@@ -0,0 +1,215 @@
+---
+title: Get an outgoing payment grant without recipients
```

Comment:
```suggestion
title: Get an outgoing payment grant for future payments
```
Because technically all of our the outgoing payment grant requests that we have documented do not correspond to a recipient